src/playground/App.js

```diff
@@ -6,54 +6,92 @@ import CrashAlert from "./components/CrashAlert";
 import Footer from "./components/Footer";
 import CodeEditor from "./components/CodeEditor";
 import { Linter, SourceCodeFixer } from "./node_modules/eslint/lib/linter/";
-import { Legacy } from "@eslint/eslintrc/universal";
 import Unicode from "./utils/unicode";
 import Configuration from "./components/Configuration";
 import Split from "react-split";
 import debounce from "./utils/debounce";
 import "./scss/split-pane.scss";
 
-const linter = new Linter();
-const rules = linter.getRules();
+const DEFAULT_TEXT = "/* eslint quotes: [\"error\", \"double\"] */\nconst a = 'b';";
+
+const linter = new Linter({ configType: "flat" });
+const legacyLinter = new Linter({ configType: "eslintrc" });
+const rules = legacyLinter.getRules();
```

Comment:
we had reached a consensus to https://github.com/eslint/eslint/issues/18103#issuecomment-1941230245.

Reply:
Agreed, but that will need some time to implement, so I think we can leave it for a follow-up. This will work until at least v10 :)

Reply:
haha, it also makes sense.